pkg/lint/runner.go

```diff
@@ -83,7 +83,7 @@ func NewRunner(cfg *config.Config, log logutils.Log, goenv *goutil.Env, es *lint
 			processors.NewIdentifierMarker(),
 
 			getExcludeProcessor(&cfg.Issues),
-			getExcludeRulesProcessor(&cfg.Issues, log, lineCache),
+			getExcludeRulesProcessor(&cfg.Issues, log, cfg.Output.PathPrefix, lineCache),
```

Comment:
It looks like path prefix and line cache get passed through different functions together. To minimize changes I had originally made "path prefix" a field in the line cache (or rather, the file cache, with an accessor in the line cache).

That didn't work because the caches got created before parsing the configuration. I would have had to add a `SetPathPrefix` to add that later, which didn't feel right.

Do you have any preference?

Reply:
In the end I went for an approach where line cache and path prefix get combined. This keeps the number of parameters that need to be passed through the same.

Reply:
Here's a diff against the earlier approach with two parameters: https://github.com/golangci/golangci-lint/compare/9664d89f477ef60739fe6ec54792b50d43099e71..d7853eaed135202c0aff13ccd6db22abe8e27d56